packages.yml

```diff
@@ -1,3 +1,3 @@
 packages:
   - package: tuva-health/the_tuva_project
-    version: [">=0.5.0","<1.0.0"]
```

Comment:
This version range looks wrong. I think it should be [">=0.6.0","<0.7.0"]

Reply:
@sarah-tuva you are reviewing the wrong diff